.tekton/scanner-db-build.yaml

```diff
@@ -68,5 +68,9 @@ spec:
         requests:
           cpu: 1
 
+  # With multi-arch the pipeline regularly takes >1h to finish
+  timeouts:
+    pipeline: 2h30m0s
```

Comment:
Oh, even DBs need such a large timeout? Asking because we don't have any compilation in dockerfiles and so the time probably goes somewhere else (waiting for a VM?)

Reply:
We've experienced some odd behaviour for the remote builds where they seem to fill the time allocated to them and we think it might be a weird interaction between the taskRunSpec resource requests and the remote builds. We're aiming to simplify this and hopefully remove these extra timeouts